Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN pip install -r ome-model/requirements.txt
 RUN mvn clean install -DskipSphinxTests
 
 WORKDIR /bio-formats-build/bioformats
-RUN ant clean jars tools test
+RUN ant jars tools
```

Comment:
Will something else run tests in travis?

Reply:
The `mvn` command above will run tests as part of the build lifecycle. Primary question is whether there is a 100% overlap with what `ant test` was executing and if not how we plan to solve this.

Reply:
Ah, ok. `skipSphinxTests` actually is `skipSphinx` not `skipSphinxAndTests`?

Reply:
Yes - see https://github.com/ome/bio-formats-documentation/blob/master/pom.xml#L522, `skipSphinxTests` actually means skip the `linkcheck` step executed as part of the `test` phase of the Sphinx component.

Reply:
Renaming it to ``skipLinkcheck`` will make sense